media/css/mozorg/moss/common.scss

```diff
@@ -0,0 +1,471 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+@import '../../pebbles/components/newsletter';
+
+
+$color-mozred-bright: saturate(lighten($color-mozred, 5%), 10%);
```

Comment:
Wonder if we should just define this in _variables.scss if we're going to use the same color in different places (home and MOSS so far).